src/Components/test/E2ETest/ServerExecutionTests/StreamingRenderingTest.cs

```diff
@@ -64,4 +64,20 @@ public void CanPerformStreamingRendering()
         Browser.FindElement(By.Id("end-response-link")).Click();
         Browser.Equal("Finished", () => getStatusText().Text);
     }
+
+    [Fact]
+    public void CanPerformFormPostWithStreamedResponses()
+    {
+        Navigate($"{ServerPathBase}/form-streaming");
+
+        // Initial "waiting" state
+        var submit = Browser.Exists(By.CssSelector("input[type=submit]"));
+        var getStatusText = () => Browser.Exists(By.Id("status"));
+        Assert.Equal("", getStatusText().Text);
+
+        submit.Click();
+
+        Assert.Equal("Processing form...", getStatusText().Text);
+        Assert.Equal("Completed", getStatusText().Text);
```

Comment:
Does this assert pattern really work? It seems like it should at least use `Browser.Equal` instead of `Assert.Equal`, and should do something explicit - such as calling some other API endpoint - to cause the processing to complete otherwise it's a race condition (maybe it will complete before you ever see the "processing" state).

Reply:
This has been replaced by proper E2E tests